packages/web-react/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -40,7 +40,7 @@ export const Breadcrumbs = <T extends ElementType = 'nav'>(props: SpiritBreadcru
                 </li>
               )}
               <li className="d-none d-tablet-flex">
-                <Icon name="chevron-right" />
+                {index !== 0 && <Icon name="chevron-right" />}
```

Comment:
Could we have a test for this, please?

Reply:
Something like "find all `li` in `ul` or `nav` a and the first item's content is not the SVG".

Reply:
@dlouhak I do not see any test or any answer...

Reply:
You're right, but then we should make tests for another test cases: presence of the element with `goBackTitle` on mobile screens, different visual and a11y properties for last breadcrumb link, etc. I would suggest create a new issue for that.

Reply:
Ok, let's move on and do it in https://github.com/lmc-eu/spirit-design-system/issues/582. Thanks.